Library/Formula/dpkg.rb

```diff
@@ -19,16 +19,23 @@ def install
     inreplace "scripts/Dpkg/Checksums.pm" do |s|
       s.gsub! "md5sum", "md5"
       s.gsub! "sha1sum", "shasum"
-      s.gsub! "sha256sum", "'shasum', '-a', '256'"
+      s.gsub! "sha256sum", "shasum', '-a', '256"
```

Comment:
unbalanced quote marks `"shasum'` and `'256"`. should this maybe just be `s.gsub! "sha256sum", "shasum -a 256"`?